src/dropdown/index.ts

```diff
@@ -352,13 +383,29 @@ export class Dropdown extends LitElement {
     this._updateContainerWidth();
   }
 
+  private _getLabelWith() {
```

Comment:
Is this the mistake of `width`?

```suggestion
  private _getLabelWidth() {
```

Reply:
ok

Reply:
i fixed it, thank you!